hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/StorageContainerManager.java

```diff
@@ -1876,16 +1879,31 @@ public boolean checkLeader() {
     }
   }
 
+  public void setScmReadOnlyAdmins(OzoneAdmins scmReadOnlyAdmins) {
+    this.scmReadOnlyAdmins = scmReadOnlyAdmins;
+  }
+
   private void checkAdminAccess(String op) throws IOException {
-    checkAdminAccess(getRemoteUser());
+    switch (op) {
+      case "startReconfiguration":
+        checkAdminAccess(getRemoteUser(), false);
+        break;
+      case "getReconfigurationStatus":
+      case "listReconfigurableProperties":
+        checkAdminAccess(getRemoteUser(), true);
+        break;
+      default:
+    }
```

Comment:
`default` branch should check write admin access.

Reply:
@adoroszlai Thanks for review, I made some changes, please review again.